taskcat/_generate_reports.py

```diff
@@ -12,136 +12,149 @@
 LOG = logging.getLogger(__name__)
 
 
+REPO_LINK = "https://github.com/aws-ia/taskcat"
+DOC_LINK = "http://taskcat.io"
+
+
 class ReportBuilder:
-    """
-    This class generates the test report.
+    """This class generates the test report."""
 
-    """
+    STYLE_SHEET_URL = "https://raw.githubusercontent.com/aws-ia/taskcat/main/assets/css/taskcat_reporting.css"
 
-    def __init__(self, stacks: Stacker, output_file: Path, version: str = "0.9.0"):
+    def __init__(self, stacks: Stacker, output_file: Path, version: str):
         self._stacks = stacks
         self._output_file = output_file
         self._version = version
 
-    # TODO: refactor for readability
```

Comment:
@andrew-glenn I may have gone too far with the refactor here. If you don't mind just giving this a quick glance to see if you want me to break it up or not. (I do still need to do a little clean up on the tests.)

As it is right now, I have made the following improvements:
- fixed HTML validation issues
- fixed version footer
- added fallback stylesheet if GH is unreachable (or transient network errors)
- added R/G colorblind helper icons
- refactored the `yattag` functions/contexts to be a bit more concise/readable (and prefered as per docs)